src/Executable.js

```diff
@@ -595,6 +604,18 @@ export default class Executable {
                 );
             }
 
+            if (this.transactionNodeIds.length) {
+                const isNodeAccountIdValid = this.transactionNodeIds.includes(
+                    nodeAccountId.toString(),
+                );
+
+                if (!isNodeAccountIdValid) {
+                    throw new Error(
+                        `Attempting to execute a transaction against node ${nodeAccountId.toString()}, which is not present in the Client's node list. Please review and adjust your setup.`,
```

Comment:
Would this still throw in the case of 1 invalid and 2 valid node account IDs being supplied? For example, if `{1, 2, 3}` node account IDs were supplied manually by the user and node account ID `1` was tried first, would this error then be thrown and execution stopped? I think we'd still want to try `2` and `3` in this case as they are still valid. Or is there processing somewhere that I missed that would make sure `1` never got selected?

Reply:
I initially considered a more restrictive approach, where an error is thrown if the client is not configured correctly, to prevent the user from making whatever he choices. However, your suggestion makes perfect sense, and I’ve updated the logic accordingly. 

Please note that with this update, we won't immediately throw a full error to the user. Instead, an error message will be logged to the console, and the process will continue with the next available node account ID. After 10 attempts, the user will be notified via a console log indicating an issue, and the error from the original flow will be thrown with the message: `"Max attempts of 10 were reached for the request, with the last error being:"``

An example of what happens when only invalid accounts are provided:
<img width="1165" alt="image" src="https://github.com/user-attachments/assets/db2e04db-b2bb-459c-b402-f6f67ad3d23c">


An example of what happens when a mix of invalid and one valid account is provided:
<img width="1129" alt="image" src="https://github.com/user-attachments/assets/7c017a78-0d0a-4b0d-a53e-a2e758e49c93">


`* You can execute the newly added integration tests to cover all the different scenarios.`

Reply:
I think this flow looks good!